app/src/main/java/com/example/happymeals/OutputListener.java

```diff
@@ -0,0 +1,8 @@
+package com.example.happymeals;
+
+public interface OutputListener {
```

Comment:
Is this output listener sitting in the main package? It should be moved to a package if so.  What is the purpose of this listener? Where should it be used? Need comments.

I suspect this listener should probably have a name change as the name "output"listener does not make sense to me. Can you justify this name?

Reply:
it's an interface I use with custom methods to check if an output from another class is expected (successful) or fails. Honestly could be removed.